htdocs/core/class/html.form.class.php

```diff
@@ -3681,6 +3681,76 @@ function select_comptes($selected = '', $htmlname = 'accountid', $statut = 0, $f
 
 		return $num;
 	}
+	
+	// phpcs:disable PEAR.NamingConventions.ValidFunctionName.NotCamelCaps
```

Comment:
This trick is to avoid warning with old code.
When we introduce new function, we try to use the camelCase syntax. So please rename your new method into
selectEstablishments
and remove disabling of phpcs.

Reply:
Ok thx eldy
done!